rnd/autogpt_server/pyproject.toml

```diff
@@ -44,6 +44,16 @@ uvicorn = { extras = ["standard"], version = "^0.30.1" }
 websockets = "^12.0"
 youtube-transcript-api = "^0.6.2"
 
+ollama = "^0.3.0"
+feedparser = "^6.0.11"
+python-dotenv = "^1.0.1"
+expiringdict = "^1.2.2"
+discord-py = "^2.4.0"
```

Comment:
I don't understand why this is marked as diff because these lines are already in `master` 🤔

Reply:
i think something went wrong resolving conflicts. let me see